<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="ipLine"/>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="portLine"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="connectBtn">
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="sendLine"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="sendBtn">
     <property name="text">
      <string>send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
